packages/volto/news/6072.feature

```diff
@@ -0,0 +1 @@
+Add support for configurable `public` directory outside Volto using `volto.config.js`. @sneridagh
```

Comment:
I have looked through the PR and I don't see where you are reading it from `volto.config.js`. If I understand correctly, it gets all public folders from all add-ons, which is good. Of course you can configure add-ons through `volto.config.js` but it's not the only way, so maybe I would clarify that it's for all add-ons IF I understood correctly :)

Reply:
I've changed the title of the PR, the commits also are missleading, so the feature only relies now in the add-ons list coming from the registry, no `volto.config.js` involved at all. This was the first shot, getting them from the addons and sum up to the build one are better indeed.

Reply:
Uh, I didn't understood initially that you were referring to the changelog entry! Sorry, now it's fixed.